.github/workflows/ci.yml

```diff
@@ -8,7 +8,6 @@ on:
   push:
     branches: [main]
   pull_request:
-    branches: [main]
```

Comment:
Not important. Just in case you are not aware. This change was done is https://github.com/microsoft/azure-privacy-sandbox-kms/pull/275/files also

Reply:
Don't worry I'm aware, I've done it in any PR that is based on #274 so that getting a CI run isn't blocked on that merging